modules/xml/src/main/java/org/mule/module/xml/transformer/XsltTransformer.java

```diff
@@ -141,6 +150,23 @@ protected void doInitialise() throws InitialisationException
         {
             throw new InitialisationException(te, this);
         }
+
+        if (logger.isWarnEnabled() && !isTransformerPoolEvictionEnabled())
+        {
+            logger.warn("Transformer pool eviction is disabled.");
+        }
+    }
+
+    /**
+     * Eviction is enabled when minEvictableIdleTimeMillis is greater than zero.
+     * @see GenericObjectPool
+     * @see GenericObjectPool#setMinEvictableIdleTimeMillis(long)
+     */
+    private boolean isTransformerPoolEvictionEnabled()
+    {
+        long timeBetweenTransformerEvictions = getTimeBetweenTransformerEvictions();
+        boolean transformerPoolEvictionIsEnabled = timeBetweenTransformerEvictions > 0;
+        return transformerPoolEvictionIsEnabled;
```

Comment:
This should be a single line. The variable names are the same as the methods that are called.